addon-SV_OptimizedListQueries.xml

```diff
@@ -22,9 +23,15 @@
   <email_templates/>
   <email_template_modifications/>
   <optiongroups>
+    <option option_id="sv_async_queries" edit_format="onoff" data_type="string" can_backup="1">
+      <default_value>0</default_value>
+      <edit_format_params></edit_format_params>
+      <sub_options></sub_options>
+      <relation group_id="performance" display_order="10140"/>
+    </option>
     <option option_id="sv_cache_nodes_guests" edit_format="spinbox" data_type="integer" can_backup="1">
       <default_value>300</default_value>
-      <edit_format_params>min=0</edit_format_params>
+      <edit_format_params></edit_format_params>
```

Comment:
Any reason to remove the lower limit on these values? Entering a negative value doesn't make sense for these